.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -1,102 +1,82 @@
 name: Bug Report
 description: File a bug report
 title: '[Bug]: '
-labels: ['Bug', 'Community backlog']
+labels: ['Bug', 'Community backlog', 'High priority']
+projects: ONS Design System project
 assignees: []
 body:
     - type: markdown
       attributes:
           value: |
               Thank you for taking the time to fill out this bug report
     - type: textarea
-      id: what-happened
+      id: describe-bug
       attributes:
-          label: What happened?
+          label: Describe the bug
+          description: 'If applicable, add screenshot(s) or examples of the bug/issue'
       validations:
           required: true
     - type: textarea
-      id: expected-outcome
+      id: expected-behaviour
       attributes:
-          label: What was the expected outcome?
-      validations:
-          required: true
-    - type: textarea
-      id: impact
-      attributes:
-          label: How is this problem impacting your service?
+          label: Expected behaviour / outcome
+          description: 'If applicable, add screenshot(s) or examples of the expected behaviour'
       validations:
           required: true
     - type: textarea
       id: reproduction-steps
       attributes:
-          label: What list of steps can be followed to reproduce the bug or issue?
+          label: List the steps necessary to reproduce the bug
       validations:
           required: true
     - type: textarea
-      id: example
+      id: additional-context
       attributes:
-          label: Example
-          description: Add screenshot(s) or examples of the bug or issue
-          placeholder: Add a file, image or link
+          label: Additional context
+          description: 'For example: describe the severity of the issue, any additional evidence, screenshots, proposed solution'
       validations:
           required: false
     - type: input
       id: version
       attributes:
           label: What version of the ONS Design System are you using?
-          placeholder: For example, 32.2.7
+          placeholder: 'For example: 32.2.7'
```

Comment:
This needs to be a label and no longer a placeholder.

Reply:
updated